extensions/twenty/src/components/Rating.tsx

```diff
@@ -0,0 +1,41 @@
+import { Form } from "@raycast/api";
+import { DataModelField } from "../services/zod/schema/recordFieldSchema";
+
+type RatingProps = {
+  field: DataModelField;
+};
+
+const Rating = ({ values }: { values: RatingProps }) => {
+  const { field } = values;
+  const defaultValue = "";
+
+  // Generate star rating options (1 to 5)
+  const options = Array.from({ length: 5 }, (_, i) => ({
+    title: "⭐️".repeat(i + 1),
+    value: `RATING_${i + 1}`,
+  }));
+
+  return (
+    <Form.Dropdown title={field.label} defaultValue={defaultValue} id={field.name}>
+      <Form.Dropdown.Item
+        key={"NO_RATING"}
+        value={""}
+        title={`No ${field.label}`}
+        icon={""}
```

Comment:
- same here: Icon option can be neglected and removed, instead of providing "", as its optional and we aren't interested in this case.
```suggestion
```